src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        Random random = new Random();
+        int randomColor = random.nextInt(5);
```

Comment:
lets make Random a class level variable - to avoid creating a new instance each time this method is invoked

avoid magic numbers - add constant for 5

Reply:
Done